pages/cart.js

```diff
@@ -0,0 +1,232 @@
+import { useContext, useEffect, useState } from "react";
+import styled from "styled-components";
+import { toast } from "react-toastify";
+import { FaTimes } from "react-icons/fa";
+
+import Header from "../components/Header";
+import Footer from "../components/Footer";
+import { Divider, Button } from "../styles/UIKit";
+import useApi from "../utils/useApi";
+import CartConstants from "../config/CartConfig";
+import AuthContext from "../contexts/AuthContext";
+
+const Cart = () => {
+  const api = useApi();
+
+  const [cartData, setCartData] = useState({});
+  const { cart, updateCart } = useContext(AuthContext);
+
+  const fetchData = () => {
+    try {
+      setCartData(cart);
+    } catch (err) {
+      toast.error("Cannot fetch cart data");
+      console.log("Error while fetching cart data : ", err);
+    }
+  };
+
+  const handleDeleteCartItem = (itemId) => {
+    deleteCartItem(itemId);
+  };
+
+  const deleteCartItem = async (itemId) => {
+    try {
+      const {
+        data: { cart },
+      } = await api.delete(`cart/product/${itemId}`);
+
+      setCartData(cart);
```

Comment:
the cart object already has the required data, so this state isn't necessary

Reply:
taken care